README.md

```diff
@@ -0,0 +1,7 @@
+# HTML FORMS Microverse Project
+
+This is the second project on the Microverse Main Curriculum list
+
+##Authors
+* Santiago Guerra
+* Vashira Musa
```

Comment:
#### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), live link (from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive.